tests/em/static/test_DC_Utils.py

```diff
@@ -326,9 +336,14 @@ def test_convert_to_2d(self):
             survey_2d.locations_n,
         ]
 
+        # Coordinates should be roughly the same
         passed = np.allclose(loc3d[:, 0::2], loc2d)
         self.assertTrue(passed)
 
+        # Check that the first x-coordinate for electrode A is zero for both surveys
+        for i in range(0, len(survey_2d_list)):
+            self.assertEqual(survey_2d_list[i].locations_a[0, 0], 0)
```

Comment:
A minor improvement on the code style here, just to make it more Pythonic.
```suggestion
        for survey in survey_2d_list:
            self.assertEqual(survey.locations_a[0, 0], 0)
```